site/content/contribute/server/developer-setup/windows-wsl.md

```diff
@@ -2,11 +2,26 @@ This is an unofficial guide. Community testing, feedback and improvements are we
 
 1. Install the Windows Subsystem for Linux: https://docs.microsoft.com/en-us/windows/wsl/install-win10
 
+      **Note:** Docker for Windows expects path to have the format `/c/foo/bar`, but WSL uses `/mnt/c/foo/bar` instead.
+      
+      Run `winver` and check which version of Windows you have. If you are using `1803` or higher, then tou need to create a file `/etc/wsl.conf` with the following content to make sure your drives are mounted at the root rather than inside `/mnt`:
+      
+       [automount]
+       root = /
+       
+      If you are using an older version, you need to create custom bind points:
+      
+       sudo mkdir /c
+       sudo mount --bind /mnt/c /c
+
+      For more on how to properly set up Docker for Windows and WSL see [here](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly).
```

Comment:
I'm not sure about linking to some random blog on internet, how are we going to be sure it'll be around for a long time?

Reply:
Well, he's a Docker Captain (https://www.docker.com/community/captains) so that's as close as we can get to something official, other than the official documentation.

Anyway, the most relevant information is here, I just wanted to offer additional details to those interested without adding too many details here, and assumed this was the way to go as there's already a link to a post on Medium in this same document.

Reply:
yeah, i saw the medium post, and I think the same but it was already there.
maybe we can try and reach a compromise linking the archive.org page, which hopefully will outlast the blog: https://web.archive.org/web/20200402131133/https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly
that way we are certain it would exist as you saw when writing docs